x-pack/plugins/uptime/public/components/monitor/ml/__snapshots__/ml_manage_job.test.tsx.snap

```diff
@@ -1,5 +1,128 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Manage ML Job rendering renders without errors 1`] = `
```

Comment:
Can we please avoid snapshot tests, we discourage these in uptime and prefer using RTL.

Reply:
Great shout, these were actually not supposed to be here as I deleted the old snapshot tests. I'll clean up.